### DISCUSSION TITLE
Multiple users?

### DISCUSSION THREAD

**Discussion by GD-Dal** [Yooooomi/your_spotify#370] [Q&A]
Very nice program.
I set up a container running this, but the question is:
Can several users connect to and use the same container / app?

Thanks

**Comment by Yooooomi** [comment]
Hello! Thanks, and yes you can, I think it's written in the readme, you have to register them in the application you created on the Spotify dashboard.
The 8th point here https://github.com/Yooooomi/your_spotify?tab=readme-ov-file#creating-the-spotify-application

**Reply by GD-Dal** [reply]
Thanks for answering.
But will they be able to see their own stats?
I figure they need to generate API keys in their own Spotify Dashboard, and add them to my docker run file

**Reply by Yooooomi** [reply]
You just need to add their account to your application dashboard then, by logging to your Spotify with their Spotify account, they will by able to have their **own** stats.

**Comment by GD-Dal** [comment]
Sorry for the spoonfeeding, but I'm assuming you mean I add users at developer.spotify.com, and not in your app?
Here?
![image](https://github.com/Yooooomi/your_spotify/assets/59178483/cc8eab2c-0ad8-4fba-8a46-b7e71bcbc6b5)

**Reply by Yooooomi** [reply]
Yes here :)
